packages/styling/src/styles/DefaultSerifs.ts

```diff
@@ -0,0 +1,11 @@
+import { ISerifs } from '../interfaces/index';
+
+export const DefaultSerifs: ISerifs = {
+  depth4: { boxShadow: '0 0 0 0 transparent' },
+  depth8: { boxShadow: '0 3.2px 7.2px 0 rgba(0, 0, 0, 0.132), 0 0.6px 1.8px 0 rgba(0, 0, 0, 0.108)' },
+  depth16: { boxShadow: '0 3.2px 7.2px 0 rgba(0, 0, 0, 0.132), 0 0.6px 1.8px 0 rgba(0, 0, 0, 0.108)' },
+  depth64: { boxShadow: '0 25.6px 57.6px 0 rgba(0, 0, 0, 0.22), 0 4.8px 14.4px 0 rgba(0, 0, 0, 0.18)' },
+
+  roundedCorner2: 2,
+  roundedCorner4: 4
```

Comment:
Rounded corner values are made-up for now, I couldn't find the real values.

Reply:
I've only seen 2px rounded corners in the Fluent designs so far.

Reply:
2px is correct.